library/src/commonMain/kotlin/com/composegears/tiamat/NavController.kt

```diff
@@ -202,25 +212,33 @@ class NavController internal constructor(
         }
     }
 
-    internal fun reset() {
+    internal fun toSavedState() = currentNavEntry?.let {
+        mapOf(
+            KEY_UUID to uuid,
+            KEY_CURRENT to currentNavEntry?.apply { saveState() }?.toSavedState(storageMode),
```

Comment:
Maybe better to use `it` from let block, to minimize null checks

Reply:
done